definitions/tools/replace_vcf_sample_name.cwl

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env cwl-runner
+
+cwlVersion: v1.0
+class: CommandLineTool
+label: "Replace the sample name in a gzipped vcf and return"
+baseCommand: ['/bin/bash', 'replacer.sh']
```

Comment:
For portability, it might be nice to accept both gzipped and non-gzipped vcfs, since we have both at various points in our pipelines.

It also might be nice to just go ahead and index the vcf in here instead of having the additional index step in the calling workflow.

Reply:
Right now this step is run twice in certain parts of the pipeline (once for the tumor and once for the normal sample name) so indexing after each step doesn't seem optimal.